services/src/main/java/org/keycloak/services/resources/SessionCodeChecks.java

```diff
@@ -229,6 +229,10 @@ public AuthenticationSessionModel initialVerifyAuthSession() {
                 }
                 event.error(Errors.ALREADY_LOGGED_IN);
             } else {
+                LoginFormsProvider loginForm = session.getProvider(LoginFormsProvider.class).setAuthenticationSession(authSession)
+                        .setError(Messages.COOKIE_NOT_FOUND);
+                response = loginForm.createInfoPage();
+                event.detail(Details.REDIRECTED_TO_CLIENT, "true");
```

Comment:
This looks strange as there is error `REDIRECTED_TO_CLIENT`, but there was not redirect to client.